### PR TITLE
IScp330Event

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Review by Misfiy — APPROVED** [review]

File: EXILED/Exiled.Events/EventArgs/Scp330/DroppingScp330EventArgs.cs [hunk 1]
```diff
@@ -39,9 +39,12 @@ public DroppingScp330EventArgs(Player player, Scp330Bag scp330, CandyKindID cand
         }
 
         /// <summary>
-        /// Gets or sets a value representing the <see cref="Item" /> being picked up.
+        /// Gets or sets a value representing the <see cref="API.Features.Items.Item" /> being picked up.
         /// </summary>
-        public Scp330 Scp330 { get; set; }
+        public Scp330 Scp330 { get; set; } // Todo Remove set
```
Comment (ITeMbI4) [inline EXILED/Exiled.Events/EventArgs/Scp330/DroppingScp330EventArgs.cs:44]: // Todo Remove set
=)

**Review by Jesus-QC — DISMISSED** [review]
Fix weird files being deleted, the rest seems fine

File: EXILED/Exiled.Events/EventArgs/Scp330/DroppingScp330EventArgs.cs (hunk 1 above)
Comment (Jesus-QC) [reply EXILED/Exiled.Events/EventArgs/Scp330/DroppingScp330EventArgs.cs:44]: The setter is needed

Comment (louis1706) [reply EXILED/Exiled.Events/EventArgs/Scp330/DroppingScp330EventArgs.cs:44]: > The setter is needed

Why ??

**Review by Vladislav-CS — APPROVED** [review]

**Pull request merged**